### PR TITLE
Support for lineCap and lineDash pattern

### PR DESCRIPTION
**Pull request by kruyvanna**
### Does any other open PR do the same thing?

Yes. But this one does one more thing which is LineDashPattern
(please answer here)

### What issue is this PR fixing?
No issue reported yet.
(please link the issue here)

### How did you test this PR?

Tested.

You're welcome

### CODE REVIEW

**Comment by christopherdro** [comment]
@kruyvanna Thanks for the PR. Any chance you can add this information to the docs?

**Comment by kruyvanna** [comment]
@christopherdro I updated the docs.

File: lib/android/src/main/java/com/airbnb/android/react/maps/AirMapPolylineManager.java [hunk 1]
```diff
@@ -97,6 +98,12 @@ public void setlineCap(AirMapPolyline view, String lineCap) {
     view.setLineCap(cap);
   }
 
+  @ReactProp(name = "lineDashPattern")
+  public void setLineDashPattern(AirMapPolyline view, ReadableArray patternValues) {
+      Log.i("VANNA", "setLineDash " + patternValues);
```
Comment (christopherdro) [inline lib/android/src/main/java/com/airbnb/android/react/maps/AirMapPolylineManager.java]: @kruyvanna Could you remove this log here?

**Comment by christopherdro** [comment]
@kruyvanna Thank you. Just left one more comment to remove the a debug log left behind.

**Comment by kruyvanna** [comment]
@christopherdro done.

File: lib/android/src/main/java/com/airbnb/android/react/maps/AirMapPolylineManager.java [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import android.graphics.Color;
 import android.os.Build;
 import android.util.DisplayMetrics;
+import android.util.Log;
```
Comment (salah-ghanim) [inline lib/android/src/main/java/com/airbnb/android/react/maps/AirMapPolylineManager.java]: @kruyvanna please remove this import

once done I'm more than happy to test and merge

**Comment by kruyvanna** [comment]
@salah-ghanim done

**Pull request merged**